index.bs

```diff
@@ -861,6 +861,7 @@ The {{MLContext}} interface represents a global state of neural network compute
 
 <script type=idl>
 typedef record<USVString, ArrayBufferView> MLNamedArrayBufferViews;
+typedef record<DOMString, MLTensor> MLNamedTensors;
```

Comment:
Can we use `USVString` instead?

(I'm guessing this is from long iteration on the MLBuffer explainer, which predated the spec adopting USVString in 3a916f26feed501467f37110db5754e8de73151a)